pnpm-workspace.yaml

```diff
@@ -1,4 +1,3 @@
 packages:
   - 'reloaded'
   - 'experiments/billable'
-  - 'experiments/cutl'
```

Comment:
@harryhcs I needed to remove this from here now that the vite plugin no longer should have that patch applied to it. Best explained over video, can you take a look when you have a moment? https://www.loom.com/share/109deb7365da476cafd73ab709da365b